templates/web/smidsy/report/new/notes.html

```diff
@@ -0,0 +1,23 @@
+[%# This is not displayed on the live site!
```

Comment:
Then why not delete this?

Reply:
I couldn't tell from any of the history whether it was a mistake that this wasn't being shown anywhere :-/